vipps-order-management-api.md

```diff
@@ -117,33 +117,33 @@ and we recommend all customers with direct integration with the API to also do s
 
 # API summary
 
-See the API reference for details:
-* [Swagger UI](https://vippsas.github.io/vipps-order-management-api/)
-* [ReDoc](https://vippsas.github.io/vipps-order-management-api/redoc.html)
+## Links
+
+TODO!
 
 ## Images
 
-[`POST:/order-management/v1/image`](https://vippsas.github.io/vipps-order-management-api/#/Image/postImage)
+[`POST:/order-management/v1/images`](https://vippsas.github.io/vipps-order-management-api/#/images/postImage)
 
 Endpoint for uploading pictures of products as a base64 string, along with an
 ID defined by the merchant. An image exists independently of any transaction.
 It is not possible to overwrite an image.
 
 ## Receipts
 
-[`POST:/order-management/v1/receipt/{vippsTransactionId}`](https://vippsas.github.io/vipps-order-management-api/#/Receipt/postReceipt)
+[`POST:/order-management/v1/receipts/{vippsTransactionId}`](https://vippsas.github.io/vipps-order-management-api/#/receipts/postReceipt)
```

Comment:
Do we have any plan for supporting getting or posting of multiple receipts at the same time? (its not supported now)

Just seems strange to have all the endpoints in "plural" when listing and multiple posts arent supported.

Reply:
nvm, vipps api rfc tells us to use plural.
https://github.com/vippsas/vipps-rfc/blob/master/0001-api-guidelines.md#use-plural